adafruit_pyportal.py

```diff
@@ -784,13 +786,14 @@ def fetch(self, refresh_url=None):
             return values[0]
         return values
 
-    def show_QR(self, qr_data, qr_size=1, x=0, y=0):  # pylint: disable=invalid-name
+    def show_QR(self, qr_data, qr_size=1, x=0, y=0, hide_background=False):  # pylint: disable=invalid-name
```

Comment:
```suggestion
    def show_QR(self, qr_data, *, qr_size=1, x=0, y=0, hide_background=False):  # pylint: disable=invalid-name
```

This should fix the lint check for arguments by making the keywords keyword-only. It may break existing code but it will make them more readable.

Reply:
Yes -- this should be fixed now. Thanks for your patience and explanations of everything. It's hard being a beginner!